corehq/apps/export/models/new.py

```diff
@@ -954,7 +959,7 @@ def generate_schema_from_builds(cls, domain, app_id, identifier, force_rebuild=F
                 identifier,
             )
 
-            current_schema.record_update(app_id, app.version)
+            current_schema.record_update(app.copy_of or app_id, app.version)
```

Comment:
for case exports this was completely busted since we'd look at apps in all domains not just the one passed in